src/Microsoft.OpenApi.OData.Reader/Generator/OpenApiInfoGenerator.cs

```diff
@@ -93,5 +98,26 @@ private static string GetDescription(this ODataContext context)
             // so a default description should be provided if no Core.LongDescription annotation is present.
             return "This OData service is located at " + context.Settings.ServiceRoot.OriginalString;
         }
+
+        private static Dictionary<string, IOpenApiExtension> GetExtensions(this ODataContext context)
+        {
+            Debug.Assert(context != null);
+
+            if (context.Settings.ShowAssemblyInfo)
+            {
+                return new Dictionary<string, IOpenApiExtension>()
+                {
+                    {
+                        "x-generatedBy",
```

Comment:
Is this extension defined anywhere? Otherwise should we add a -ms and normalize to kebab casing?

Reply:
I can't find it. I'll rename it.